src/content/docs/ecosystem/validators/watchdog/apis.mdx

```diff
@@ -22,12 +22,12 @@ description: SKALE Watchdog APIs
 
 == Introduction
```

Comment:
Change all == on this page to ## -> Not sure why this didn't get updated during migration